Add Peacock case with 5 deg tilt

kennnyshiwa opened the pull request:
Adds a second case option with  a 5 degree tilt for better ergonomics

kennnyshiwa committed:
Add Peacock case with 5 deg tilt

george-norton commented:
Thanks! Any chance you could add a short note in the readme file, perhaps with a picture of you have one?

george-norton commented:
Also, probably not an issue, but I uploaded the STEP file to JLC and they are worried about some thin walls:
![image](https://github.com/user-attachments/assets/62ddd5f4-353f-4e1d-b5c3-928ebc26dc1b)

kennnyshiwa committed:
Add image of the new case type

kennnyshiwa commented:
Added in a small note on the readme along with a photo of the new case, 

The chamfer that I added to get the 5 deg tilt does cause that front edge to become a tad thin. However from all of the prints I've made so far it has not been an issue, Some filaments can cause it to be a little more transparent along the very bottom edge, but trustfully the case is still very strong and not at risk of failure on a traditional FDM printer. If this were being machined I'd caution against it but for traditional prints it appears to be fine still.

george-norton committed:
Fix image link
